customizations/mei-CMN.xml

```diff
@@ -105,6 +105,21 @@
           </content>
         </moduleRef>
 
+        <classSpec ident="att.meiVersion" module="MEI.shared" type="atts" mode="change">
+          <attList>
+            <attDef ident="meiversion" usage="rec" mode="change">
+              <valList type="closed">
+                <valItem ident="5.0-dev">
```

Comment:
As this is on the `release/5.0` branch, it's probably safe to just change this to `5.0` directly, and remove the `-dev`.

Would save an additional PR, and would give us a bit more time to find any weird issues that might arise when changing this.